emission/tests/funcTests/TestFunctionTiming.py

```diff
@@ -35,207 +17,114 @@ def setUpClass(self):
-        logging.info(
-            f"'{name}': Expected elapsed time = {expected_elapsed_ms:.2f} ms, "
-            f"Stored reading = {stored_reading} ms."
-        )
+        logging.debug(f"Ensuring {len(expected_entries)} entries exist in DB.")
+        key_list = [key for (key, _, _) in expected_entries]
+        stored_entrys = list(self.timeseries_db.find_entries(key_list))
+        self.assertEqual(len(stored_entrys), len(expected_entries))
 
-        # Assert that the stored_reading matches elapsed_ms exactly
-        self.assertEqual(
-            stored_reading,
-            expected_elapsed_ms,
-            msg=(
-                f"Stored reading {stored_reading} ms does not exactly match "
-                f"elapsed time {expected_elapsed_ms:.2f} ms for '{name}'."
-            )
-        )
-
-        # Assert that the stored_name matches name exactly
-        self.assertEqual(
-            stored_name,
-            name,
-            msg=(
-                f"Stored name '{stored_name}' does not match function "
-                f"name '{name}'."
-            )
-        )
+        for i in range(len(expected_entries)):
+            stored_entry = stored_entrys[i]
+            expected_key, expected_name, expected_reading = expected_entries[i]
+            logging.debug(f"Comparing expected {expected_entries[i]} " +
+                          f"with stored {stored_entry['metadata']['key']} {stored_entry['data']}")
+            self.assertEqual(stored_entry['metadata']['key'], expected_key)
+            self.assertEqual(stored_entry['data']['name'], expected_name)
+            self.assertEqual(stored_entry['data']['reading'], expected_reading)
 
     def test_single_function_timing(self):
         """
         Test execution and timing of a single function.
         """
         def sample_function():
-            logging.info("Executing sample_function")
+            logging.debug("Executing sample_function")
             time.sleep(2)  # Simulate processing time
             return True
 
-        with test_logger("test_single_function_timing"):
-            self.run_with_timing("sample_function", sample_function)
-
-    def test_multiple_functions_timing(self):
-        """
-        Test execution and timing of multiple functions within a single test.
-        Ensures that each function's timing is recorded separately.
-        """
-        def function_one():
-            logging.info("Executing function_one")
-            time.sleep(1)
-            return True
-
-        def function_two():
-            logging.info("Executing function_two")
-            time.sleep(1.5)
-            return True
-
-        functions = [function_one, function_two]
+        with ect.Timer() as timer:
+            sample_function()
 
-        with test_logger("test_multiple_functions_timing"):
-            for func in functions:
-                self.run_with_timing(func.__name__, func)
+        esdsq.store_dashboard_time("sample_function", timer)
```

Comment:
This is the way you're going to actually use `store_dashboard_time` (on `op-admin-dashboard`), right?
So I recommend that for the purpose of testing, you use the same pattern in each test.